src/main/java/lv/venta/fitness/services/IMealService.java

```diff
@@ -3,14 +3,21 @@
 import java.util.ArrayList;
 import java.util.Collection;
 
-import lv.venta.fitness.models.Ingredient;
 import lv.venta.fitness.models.Meal;
 
 public interface IMealService {
 	ArrayList<Meal> selectAllMeals();
 	ArrayList<Meal> selectMealsWithCaloriesOver(int calories) throws Exception;
 	ArrayList<Meal> selectMealsWithCaloriesUnder(int calories) throws Exception;
 	ArrayList<Meal> selectMealsWithProteinOver(int protein) throws Exception;
-	void insertNewMeal(String title, String description, Collection<Ingredient> ingredients) throws Exception;
+
+	void insertNewMeal(String title, String description, int calories, int fat, int carbohydrates, int protein) throws Exception;
+
 	void deleteMealById(long idme) throws Exception;
+
+	Meal insertEmptyMealEntry(long idhe);
+
+	ArrayList<Meal> getMealsByHealthId(long idhe) throws Exception;
+
+	void updateMealById(long idme, Meal data) throws Exception;
```

Comment:
Return int:
1 if successful
-1 if unsuccessful
Aka just returns in the try {} catch {} statemant